### PR TITLE
Accept directories

### PR DESCRIPTION
**Pull request by airtonzanon**

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -8,57 +8,91 @@ const packageVersion = require("../package").version;
...
+    console.log(wrongFiles);
     process.exit(1);
   }
-
+  
+  !program.fix || console.log(fixedFiles);
   process.exit(0);
 }
 
+function readDirectory(directoryPath) {
```
Comment (vicnicius) [inline lib/cli.js]: This function is reading files AND processing files. I'd suggest you either rename it or stick to making it just "read" the directory and then get the output with the full file path through the processFile function. Something like:
```js
function readDirectory(dir) {
  ...
  return dir + fileName;
}
...
readDirectory(dirPath).map(processFile); 
```

**Comment by vicnicius** [comment]
No tests?

**Comment by airtonzanon** [comment]
@vicnicius habemus tests

File: lib/cli.js [hunk 2]
```diff
@@ -5,17 +5,13 @@ const path = require("path");
...
+const { getFiles, fixJsonFile, showDiff } = require("./modules.js");
 
 const program = new commander.Command();
-const blue = "\x1b[34m";
-const red = "\x1b[31m";
-const withe = "\x1b[37m";
 
 let jsonFile = "";
-let hasErrors = false;
+let hasErrors = [];
```
Comment (vicnicius) [inline lib/cli.js]: minor: it is a bit weird a variable named hasSomething being an array.

**Pull request merged**